src/cbottle/inference.py

```diff
@@ -104,6 +122,15 @@ def __init__(
         self.time_stepper = time_stepper
         self.channels_last = channels_last
         self._convert_model_NHWC()
+        if torch_compile:
+            self.torch_compile()
+
+    def torch_compile(self):
```

Comment:
can we make this private since there is already a class level argument for it?
```suggestion
    def _torch_compile(self):
```